Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
Ok, lets have those.

'remove()' is possible, list is not, it is reserved, it has to be either 'getAll()' or 'getBackups()'.

Do you agree to have both -> Iterator/Countable + getBackups() => it is practically the same.